test/fail_compilation/fail4611.d

```diff
@@ -1,16 +0,0 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail4611.d(15): Error: Vec[1000000000] size 4 * 1000000000 exceeds 16MiB size limit for static array
----
-*/
-
-struct Vec
-{
-    int x;
-}
-
-void main()
-{
-    Vec[1000_000_000] a;
-}
```

Comment:
I think you either need to modify the test runner or use a shell script instead, like this: https://github.com/dlang/dmd/blob/master/test/runnable/test16096.sh#L3-L6.